### ISSUE TITLE
Incorrect chronological book order in some translations.

### ISSUE THREAD

**Issue by akdevservices** [Bolls-Bible/bain#23]
Incorrect chronological book order in some translations.

Наприклад, переклад Огієнка:
![image](https://github.com/user-attachments/assets/76e1bc56-a7fe-4e24-8ea3-b08275d157bf)

Переклад Хоменка:
![image](https://github.com/user-attachments/assets/ea05f9a3-cc0f-4b71-b5e8-238f29caa871)

Якщо увімкнути та знову вимкнути опцію `Chronological order`, то отримуємо очікуваний (правильний) стандартний (нехронологічний) порядок книг Біблії.

Переклад Хоменка (для перекладу Огієнка те саме):
![image](https://github.com/user-attachments/assets/2f52d873-aedc-4f53-bc21-63224665a439)

Але якщо перезавантажити браузер, то отримуємо знову некоректний порядок книг для вимкненої опції `Chronological order`.

**Comment by Bohooslav** [comment]
Там була проблема суто з перекладом Хоменка як я зрозумів, в json воно було посортоване не по bookid.

**Comment by akdevservices** [comment]
Переклад Огієнка UBIO має схожу проблему. Можливо інші теж. Просто я користуюсь перекладом Огієнка і тому помітив.

**Comment by Bohooslav** [comment]
Гляну ще раз...

**Comment by akdevservices** [comment]
Зараз подивився, переклад Хоменка виглядає як треба, а Огієнка все ще має проблему.

**Comment by Bohooslav** [comment]
Тікишо пушнув нову версію v2.5.4, має робити все однаково.

**Comment by akdevservices** [comment]
Наче працює, але не повністю :)

Steps to reproduce:
1. Clear cookies and site data.
2. Reload https://bolls.life/ (got YLT translation by default)
3. Switch to UBIO translation, see the incorrect book order with `Chronological Order` option disabled by default (see the screenshot below).

![image](https://github.com/user-attachments/assets/08a2958f-ff72-4ac2-b39f-6a5bb75513bf)

Що цікаво, якщо увімкнути та вимкнути опцію `Chronological Order`, то ми отримуємо коректний порядок книг, який вже зберігається після перезавантаження браузера.

**Comment by Bohooslav** [comment]
Ах да, забув за той момент. Ви випадком не QA 😅?

**Comment by akdevservices** [comment]
Ні, я розробник, а розробників колишніх не буває. Якщо відверто, я особливо не крутив ні веб, ні додаток, просто користуюсь, а підняв тільки те, що заважає або муляє. Працює і добре. Ще раз дякую за зручні біблійні додатки, які роблять саме те, що треба - несуть Слово Боже, а не соціальні функції та рекламу.

**Comment by akdevservices** [comment]
> Ах да, забув за той момент.

Цей баг досі присутній :)
